src/rng.rs

```diff
@@ -76,7 +76,7 @@ impl<D: Digest> FiatShamirRng for FiatShamirChaChaRng<D> {
     }
 
     /// Create a new `Self` by initializing with a fresh seed.
-    /// `self.seed = H(self.seed || new_seed)`.
+    /// `self.seed = H(self.seed)`.
```

Comment:
I think it was correct like before no ?

Reply:
hm, not excactly. The function uses a digest/hash to "normalize" the seed to a 32 bytes, which are used to initialize the ChaCha rng. But the comment now is misleading, too :)
Let us simply erase line 79.

Reply:
Done